src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -342,6 +327,28 @@ private async Task OnCircuitDownAsync(CancellationToken cancellationToken)
             }
         }
 
+        // Called by the client when it completes rendering a batch.
+        // OnRenderCompleted is used in a fire-and-forget context, so it's responsible for its own
+        // error handling.
+        public async Task OnRenderCompleted(long renderId, string errorMessageOrNull)
```

Comment:
`OnRenderCompleteAsync` please, the rest of the methods in this class are using that convention.

Reply:
done